README.md

```diff
@@ -96,6 +96,7 @@ Please take a look at the [Contribution Guidelines](https://github.com/meilisear
 * [Laravel](https://github.com/laravel/scout) - Official Laravel-Scout package
 * [Ruby on Rails](https://github.com/meilisearch/meilisearch-rails)
 * [Symfony](https://github.com/meilisearch/meilisearch-symfony)
+* [Nuxt](https://github.com/xlanex6/nuxt-meilisearch)
```

Comment:
Hey @xlanex6 thanks so much for adding this (and your work).
It won't be part of the `Official integrations`, can you move in to the `Community integrations` section?
Adding a small desc would help (like `a module for Nuxt `- _is this called module in Nuxt ecosytem?_)

Reply:
Hey @ferdi05 

Of course I can move to Community section . I even notice I published on `Official` one
I do it now